examples/image_processing/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ project(ArrayFire-Example-Image-Processing
 
 find_package(ArrayFire)
 
-add_definitions("-DASSETS_DIR=\"${ASSETS_DIR}\"")
+add_definitions(-DASSETS_DIR=\"${ASSETS_DIR}\")
```

Comment:
I don't know about this. I was almost certain you needed to quotes for CMake to handle the escape sequence. Can you make sure this is working by examining the command line?

Reply:
It is working fine without quotes. In fact, I only made this consistent another file where quotes weren't used earlier. So, I tested it without quotes and it worked fine.

Reply:
Just realised, without quotes, windows causes issues if the path has spaces.windo